### ISSUE TITLE
[please re-test] Safari Firefox: Remove Focus from Disabled Checkout Steps

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4339]
Disabled checkout steps are receiving focus, this is a bug. User should be able to TAB back to steps already visited, but shouldn't be able to TAB forward to steps that are disabled.
**Disabled** steps should be made **non-focusables**.

![Screen Shot 2019-08-29 at 9 25 52 PM](https://user-images.githubusercontent.com/15113219/63986945-2d010680-caa4-11e9-869e-af9baddc735b.png)
![Screen Shot 2019-08-29 at 9 26 19 PM](https://user-images.githubusercontent.com/15113219/63986951-31c5ba80-caa4-11e9-9f6d-ae22dcf3efc9.png)
![Screen Shot 2019-08-29 at 9 26 52 PM](https://user-images.githubusercontent.com/15113219/63986955-34281480-caa4-11e9-8eb3-845b2ae8c719.png)

**Comment by KateChuen** [comment]
Chrome OK.

On firefox and safari, user cannot tab back to active(previously visited) checkout steps.

**Comment by developpeurweb** [comment]
Seems to be fixed now, I'm able to tab backwards (`Shift` + `TAB` keys) pls re-test.

**Comment by KateChuen** [comment]
Tested on dev3.
chrome ✅
firefox ✅
safari ✅
